.github/workflows/buildAndTestCMake.yml

```diff
@@ -94,3 +94,8 @@ jobs:
           CMAKE_BUILD_TYPE: Release
           STABLEHLO_ENABLE_BINDINGS_PYTHON: ON
           STABLEHLO_ENABLE_PYTHON_TF_TESTS: ON
+
```

Comment:
consider changing `_toc.yaml` to point at the new pages.. I'm not sure how this will work in practice so can do after, but I'd assume it'll just work?

Reply:
Thanks for pointing that out, I was planning to do that in a follow-up if that's okay.